store-implementation/map-store/src/main/java/uk/gov/gchq/gaffer/mapstore/impl/GetElementsUtil.java

```diff
@@ -99,12 +99,13 @@ private GetElementsUtil() {
         } else {
             relevantElements = new HashSet<>();
 
-            final EdgeId edgeId = (EdgeSeed) elementId;
```

Comment:
This is the cast that was causing errors. Now we create a new `EdgeSeed` below, whether from an `EdgeSeed` or an `Edge`.